dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/UpdateReport.java

```diff
@@ -40,9 +39,9 @@ public static void main(String[] args) throws RepositoryException {
     
     try {
       DefaultArtifact artifact = new DefaultArtifact(args[0]);
-      
-      DependencyGraph graph =
-          DependencyGraphBuilder.getCompleteDependencies(artifact);
+
+      DependencyGraphBuilder dependencyGraphBuilder = new DependencyGraphBuilder();
```

Comment:
There's no configuration here, which suggests this should still be static.

Reply:
The reason of making this non-static is to hold internal state `artifactProblems`.

Reply:
Implementation detail. This should not change the public API.